<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1662</width>
    <height>943</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Foundry</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QFrame" name="logFrame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="logLabel">
          <property name="text">
           <string>Log</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="logTextEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="opBox"/>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="viewportsFrame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
         <horstretch>4</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>800</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="viewportsLayout"/>
      </widget>
      <widget class="QFrame" name="userListFrame">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="userListLabel">
          <property name="text">
           <string>Users</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="userListView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1662</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewProject"/>
    <addaction name="separator"/>
    <addaction name="actionOpenProject"/>
    <addaction name="actionSaveProject"/>
    <addaction name="actionSaveProjectAs"/>
    <addaction name="separator"/>
    <addaction name="actionImportModels"/>
    <addaction name="separator"/>
    <addaction name="actionFabricate"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionScaleDialog"/>
    <addaction name="actionRotateDialog"/>
    <addaction name="actionCenterOnPlatform"/>
    <addaction name="separator"/>
    <addaction name="actionSelectAll"/>
    <addaction name="actionSelectNone"/>
   </widget>
   <widget class="QMenu" name="menuBricks">
    <property name="title">
     <string>Bricks</string>
    </property>
    <addaction name="actionCreate_Brick"/>
   </widget>
   <widget class="QMenu" name="menuDeveloper">
    <property name="title">
     <string>Developer</string>
    </property>
    <addaction name="actionDumpScenegraph"/>
    <addaction name="separator"/>
    <addaction name="actionNetwork"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuBricks"/>
   <addaction name="menuDeveloper"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionImportModels"/>
   <addaction name="separator"/>
   <addaction name="actionView"/>
   <addaction name="actionSelect"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenProject">
   <property name="text">
    <string>Open Project...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveProject">
   <property name="text">
    <string>Save Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionImportModels">
   <property name="text">
    <string>Import Models...</string>
   </property>
   <property name="toolTip">
    <string>Import Models</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionShowViewer1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Camera 1</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionShowViewer2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Camera 2</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="actionView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Tumble</string>
   </property>
  </action>
  <action name="actionShowViewer3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Camera 3</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionShowViewer4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Camera 4</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="actionSetOrthographic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Set Orthographic</string>
   </property>
  </action>
  <action name="actionSetPerspective">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Set Perspective</string>
   </property>
  </action>
  <action name="actionShowPreviewViewer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Preview Camera</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+5</string>
   </property>
  </action>
  <action name="actionRefabricate">
   <property name="text">
    <string>Update</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionTranslate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Translate</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Rotate</string>
   </property>
  </action>
  <action name="actionScale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Scale</string>
   </property>
  </action>
  <action name="actionShowSliceViewer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Slice Viewer</string>
   </property>
  </action>
  <action name="actionFastSDF">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fast SDF</string>
   </property>
  </action>
  <action name="actionDeveloper">
   <property name="text">
    <string>Developer</string>
   </property>
   <property name="toolTip">
    <string>Developer</string>
   </property>
  </action>
  <action name="actionRenderStats">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Render Stats</string>
   </property>
  </action>
  <action name="actionDumpInventor">
   <property name="text">
    <string>Dump Inventor</string>
   </property>
  </action>
  <action name="actionDumpFabGraph">
   <property name="text">
    <string>Dump FabGraph</string>
   </property>
  </action>
  <action name="actionBackfaceCulling">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Backface Culling</string>
   </property>
  </action>
  <action name="actionNewProject">
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSaveProjectAs">
   <property name="text">
    <string>Save Project As...</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect...</string>
   </property>
  </action>
  <action name="actionFabricatePreviewSurface">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Preview Surface</string>
   </property>
  </action>
  <action name="actionFabricatePreviewVolume">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Preview Volume</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionFabricatePreviewSlices">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Preview Slices</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionFabricatePreview">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Preview</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionFabricate">
   <property name="text">
    <string>Fabricate...</string>
   </property>
  </action>
  <action name="actionVerbose">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Verbose</string>
   </property>
  </action>
  <action name="actionAddImageTexture2D">
   <property name="text">
    <string>Add Texture2D</string>
   </property>
  </action>
  <action name="actionAddImageTexture3D">
   <property name="text">
    <string>Add Texture3D</string>
   </property>
  </action>
  <action name="actionReloadOps">
   <property name="text">
    <string>Reload Ops</string>
   </property>
  </action>
  <action name="actionViewEnvironment">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Environment</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionDumpFabWorld">
   <property name="text">
    <string>Dump FabWorld</string>
   </property>
  </action>
  <action name="actionViewOperatorGuide">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Operator Guide</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionViewOperatorManipulator">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Operator Manipulator</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionViewSliceGuide">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Slice Guide</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="actionTiming">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Timing</string>
   </property>
  </action>
  <action name="actionFabGraphEnable">
   <property name="text">
    <string>Enable</string>
   </property>
  </action>
  <action name="actionFabGraphDisable">
   <property name="text">
    <string>Disable</string>
   </property>
  </action>
  <action name="actionFabGraphEnableAll">
   <property name="text">
    <string>Enable All</string>
   </property>
  </action>
  <action name="actionDithering">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dithering</string>
   </property>
  </action>
  <action name="actionAddObjectGenerator">
   <property name="text">
    <string>Add Object Generator</string>
   </property>
  </action>
  <action name="actionFabricateCrop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Crop</string>
   </property>
  </action>
  <action name="actionFabricateStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionScaleDialog">
   <property name="text">
    <string>Scale...</string>
   </property>
  </action>
  <action name="actionRotateDialog">
   <property name="text">
    <string>Rotate...</string>
   </property>
  </action>
  <action name="actionCenterOnPlatform">
   <property name="text">
    <string>Center on Platform</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionSelectNone">
   <property name="text">
    <string>Select None</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+D</string>
   </property>
  </action>
  <action name="actionDrawStyleWireframe">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Wireframe</string>
   </property>
   <property name="toolTip">
    <string>Set draw style to wireframe</string>
   </property>
   <property name="shortcut">
    <string>Meta+2</string>
   </property>
  </action>
  <action name="actionDrawStylePolygons">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Polygons</string>
   </property>
   <property name="toolTip">
    <string>Set draw style to polygons</string>
   </property>
   <property name="shortcut">
    <string>Meta+1</string>
   </property>
  </action>
  <action name="actionDrawStylePoints">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Points</string>
   </property>
   <property name="toolTip">
    <string>Set draw style to points</string>
   </property>
   <property name="shortcut">
    <string>Meta+3</string>
   </property>
  </action>
  <action name="action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Box Slice</string>
   </property>
  </action>
  <action name="actionBoxSlice">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable Box Slice</string>
   </property>
  </action>
  <action name="actionHideSliceManipulators">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hide Slice Manipulators</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionInvertSlice">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Invert Slice</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionPlaneSlice">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable Plane Slice</string>
   </property>
  </action>
  <action name="actionRotateSlice">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rotate Slice</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionWedgeSlice">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable Wedge Slice</string>
   </property>
  </action>
  <action name="actionF">
   <property name="text">
    <string>f</string>
   </property>
  </action>
  <action name="actionCam1">
   <property name="text">
    <string>Cam1</string>
   </property>
  </action>
  <action name="actionVolumePolygonTumble">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Volume Polygon Tumble</string>
   </property>
  </action>
  <action name="actionCreate_Brick">
   <property name="text">
    <string>Create Brick</string>
   </property>
  </action>
  <action name="actionDumpScenegraph">
   <property name="text">
    <string>Dump Scenegraph</string>
   </property>
  </action>
  <action name="actionNetwork">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Network</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
